sql/migrations/reports/000005_alter_reports_add_reported_at_index.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX CONCURRENTLY reports_reported_at_index ON reports (reported_at);
```

Comment:
Concurrently since introduced in Postgres 15 will not be supported in all rudder server setups.

We can run it manually with concurrently in Postgres that support it.

We should also add 'IF NOT EXISTS'  so we can run it manually first with no issues

Reply:
Made this change @lvrach . Marking this conversation as resolved